core/tests/test_window.py

```diff
@@ -370,6 +371,8 @@ def test_screen(window, app):
 
 
 def test_screen_position(window, app):
+    """A window can be moved using both the
+    absolute position and relative screen position"""
```

Comment:
The start-of-test comment should be a single line.

Reply:
Changed it